server/app/modules/CiviFormLogoutLogic.java

```diff
@@ -36,7 +43,25 @@ public Object perform(
               inputCentralLogout,
               parameters);
 
-      // TODO(#6975): Remove the session ID from the database at logout
+      // Remove the session ID from the database
+      try {
+        CallContext callContext = buildContext(config, parameters);
+        Optional<CiviFormProfile> maybeProfile =
+            profileUtils.optionalCurrentUserProfile(callContext.webContext());
+        if (maybeProfile.isPresent()) {
+          CiviFormProfile profile = maybeProfile.get();
+          var unused =
```

Comment:
(blocking) If this is required add a comment because it'll be easy to think to rename that and find errorprone breaks.